youtube-tracker/README.md

```diff
@@ -0,0 +1,51 @@
+# Track YouTube video views and comments
+
+_Tracks views, likes, and comments for provided YouTube URLs in a Google Sheet with optional email notifications._
+
+Last updated: August, 2019
+
+Track performance of public YouTube videos, including views, likes, and comments, in a Google Sheet. 
```

Comment:
This description feels very mechanical. Consider weaving a bit more story into this, talk about the problem being solved, etc. Doesn't need to be your exact story for creating it, but something that paints a better picture.

Some examples:
https://developers.google.com/gsuite/solutions/vacation-calendar
https://developers.google.com/gsuite/solutions/call-in-sick

Reply:
I am submitting the rest of the changes, will work on making this one!

Reply:
The rest looks good, let me know when this is done.

Reply:
Ok, I updated the short and long descriptions. Let me know what you think!